server/src/lib.rs

```diff
@@ -114,6 +116,26 @@ impl Modify for SecurityAddon {
     }
 }
 
+pub async fn set_up_rabbit() -> Pool {
+    let manager = Manager::new(get_env!("RABBITMQ_HOST", "RABBITMQ_HOST must be set"), ConnectionProperties::default());
+    let pool: Pool = deadpool::managed::Pool::builder(manager)
+        .max_size(10)
+        .build()
+        .expect("can create pool");
+
+    let channel = pool.get().await.expect("can get connection").create_channel().await.expect("can create channel");
+    let _ = channel
```

Comment:
```suggestion
    channel
```
Let _ is only needed if you don't expect/unwrap/or return an error, in this case we expect so its not needed